News-Android-App/src/main/res/layout/toolbar_layout.xml

```diff
@@ -1,36 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <com.google.android.material.appbar.AppBarLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <LinearLayout
+    <com.google.android.material.appbar.MaterialToolbar
```

Comment:
Switching to `MaterialToolbar` as is best practice. No further changes are needed by this switch.